crates/codegen/parser/src/to_parser_code.rs

```diff
@@ -13,19 +13,20 @@ impl<'context> CombinatorNode<'context> {
              */
             Self::Reference { tree } => match tree.production.as_ref() {
```

Comment:
I'm planning on adding `clippy` at some point later in #155, which should automate things like this:

https://rust-lang.github.io/rust-clippy/master/index.html#uninlined_format_args